src/commands/daily.ts

```
@@ -33,7 +35,9 @@ export const dailyCommand = define({
 		},
 	},
 	async run(ctx) {
-		if (ctx.values.json) {
+		// --jq implies --json
+		const useJson = ctx.values.json || ctx.values.jq != null;
```

ESLint

[error] Unsafe assignment of an `any` value.

(ts/no-unsafe-assignment)